public/img/logos/17Media.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 680.8 414.5" overflow="scroll" xml:space="preserve"><path fill="#FFFFFF" d="M326.8,0.6c28.4-2.4,57.5,2.7,83.2,15.2C452.2,35.7,484.6,75,496,120.2c9.2,35.7,5.7,74.6-10,108 c-15.9,34.1-44.2,62.2-78.4,77.8c-41.1,19-90.3,19.4-131.7,1.2c-30.1-13.1-56-35.8-73-63.9c-13.8-22.7-21.8-49-22.9-75.6 c-1.8-35.8,9-72.1,30-101.1C236.9,28.9,280.7,4,326.8,0.6L326.8,0.6z"/><path fill="#FFFFFF" d="M55.8,123.4c11.7-2.7,24.6,0.3,33.7,8.1c11.3,9.4,17.2,23.8,19.3,38c2.3,16.6,0.5,33.8-5.5,49.4 c-4.8,12.4-13.9,23.3-26.2,28.7c-15.9,7-34.4,5.5-50.3-0.4c-10.5-3.8-20.1-11.5-24-22.2C-1.5,213.1-0.2,200,2.5,188 c4.3-19.6,13.9-38.6,29.2-51.9C38.6,130.1,46.8,125.4,55.8,123.4L55.8,123.4z"/><path fill="#FFFFFF" d="M607.7,123.3c13.1-3,26.9,1.6,37.4,9.5c14.7,11,24.8,27.3,30.4,44.7c4.3,13.8,7.2,28.7,3.8,43 c-2.4,10.7-10.3,19.5-20.1,24.3c-13.3,6.4-28.6,8.9-43.2,6.6c-13.9-2.1-26.6-10.7-33.8-22.8c-8.9-14.9-11.8-32.7-11.2-49.9 c0.6-14.2,4.6-28.6,13.3-40C590.1,131.2,598.3,125.3,607.7,123.3L607.7,123.3z"/><path fill="#FFFFFF" d="M181.2,272.6c7.4-4.8,18.6,1.4,18.6,10.2c-0.9,10.3-5.9,19.9-12.8,27.4c8.9,21.9,25.9,39.7,45.7,52.1 c23,14.4,49.4,22.8,76.2,26.6c42.2,5.9,86.5,0.1,125-18.6c25.9-12.6,49.1-33.1,60-60.2c-6.8-7.5-11.9-16.9-12.8-27.1 c-0.3-9.1,11.6-15.6,18.9-10.2c4.1,2.9,3.4,8.5,5.5,12.7c3.8,9.7,13.2,16.9,23.6,17.7c3.8,0.5,8.3-0.7,11.6,2c6.3,5,5.1,16.3-2,19.9 c-2.8,1.7-6.2,1.3-9.3,1.3c-5.5,0-10.8-1.4-15.9-3.2c-12.8,29.7-38.2,52.3-66.6,66.7c-41.1,21.1-88.6,27.9-134.2,23 c-21.5-2.4-42.7-7.6-62.7-15.8c-21.7-8.9-42.2-21.5-58.6-38.5c-9.9-10.3-18.1-22.3-23.8-35.5c-7.3,2.6-15.1,3.9-22.8,2.9 c-8.9-1.9-11.6-15.1-4.7-20.8c3.5-3,8.4-1.5,12.5-2.2c10.3-1.2,19.4-8.4,23-18.1C177.3,280.9,176.9,275.3,181.2,272.6L181.2,272.6z" /></svg>
```

Comment:
Not sure how to best solve this (apart from manually redo some logos), but I believe the logos are currently being displayed against a white background, so in certain cases, these transparent+white logos will be completely invisible.

https://fx-breach-alerts.herokuapp.com/?breach=17Media (image is currently broken, but this just shows how the image will currently be rendered. You can see the "?breach=Adobe" for a non-404-ing image)